s3transfer/manager.py

```diff
@@ -586,6 +586,10 @@ def wait(self):
             # If Keyboard interrupt is raised while waiting for
             # the result, then exit out of the wait and raise the
             # exception
+            if transfer_coordinator:
```

Comment:
I will add ``transfer_coordinator = None`` just before the for loop. It could happen but is improbable. You would need to hit Cntrl-C right when that for loop was entered and the list has to be empty.